src/components/ContactInfoSummary.tsx

```diff
@@ -0,0 +1,228 @@
+import { Col, Row, Space, Typography } from 'antd';
+import dateformat from 'dateformat';
+import React from 'react';
+import styled from 'styled-components';
+import { ContactInfo } from '../containers/setContacts/ducks/types';
+import { DEFAULT_IMAGE } from '../utils/copy';
+const { Text } = Typography;
+
+interface ContactInfoSummaryProps {
+  userId: number;
+  info: ContactInfo;
+}
+
+const ContactRow = styled.div`
+  height: 200px;
+  margin-bottom: 16px;
+`;
+
+const ProfilePicture = styled.img`
+  width: 200px;
+  height: 200px;
+
+  object-fit: cover;
+  float: left;
+  margin-right: 16px;
+`;
+
+const CopyContainer = styled.div`
+  width: 100%;
+  height: 200px;
+`;
+
+const NameTitle = styled(Text)`
+  font-size: 1.6rem;
+  display: block;
+  margin-bottom: 16px;
+`;
+
+const SectionTitle = styled(Text)`
+  font-weight: 800;
+  font-size: 1.2rem;
+  display: block;
+  margin-bottom: 16px;
+`;
+
+const FamilyDetails: React.FC<ContactInfoSummaryProps> = ({ userId, info }) => {
+  return (
+    <>
+      <SectionTitle>Account Owner</SectionTitle>
+      <ContactRow>
+        <ProfilePicture src={DEFAULT_IMAGE} />
+        <CopyContainer>
+          <Row>
+            <Col span={24}>
+              <Text>User ID: {userId}</Text>
+              <NameTitle>
+                {info.mainContact.firstName} {info.mainContact.lastName}
+              </NameTitle>
+            </Col>
+          </Row>
+          <Row>
+            <Col span={6}>
+              <Space direction="vertical">
+                <span>
+                  <Text strong>Email: </Text>
+                  <Text>{info.mainContact.email}</Text>
```

Comment:
⛏️ You could also abstract this `<Text strong /><Text />` pattern into a component - not sure how often it's used elsewhere